download_scripts/change_dataset_preprocessing.py

```diff
@@ -80,10 +80,10 @@ def get_filter_on_small_documents_args(row: Dict) -> Optional[str]:
 def main():
     data = pd.read_csv("training.csv")
 
-    data["dedups"] = [" ".join(get_dedup_args(row[1])) for row in data.iterrows()]
+    data["dedups"] = data.apply(lambda row: " ".join(get_dedup_args(row)), axis=1)
     print(data[:5]["dedups"])
 
-    data["filter_short_documents"] = [get_filter_on_small_documents_args(row[1]) for row in data.iterrows()]
+    data["filter_short_documents"] = data.apply(get_filter_on_small_documents_args, axis=1)
```

Comment:
Will this still work as previously you just passed one element of the row (`row[1]`) to the `get_filter_on_small_documents_args` and now you pass full rows?

Reply:
https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.iterrows.html shows the first element is the index, which I don't need.

Reply:
ah ok that makes sense. then a better readable version of before's code would have been:
```python
get_filter_on_small_documents_args(row) for index, row in data.iterrows()
```